.github/workflows/terraform.yml

```diff
@@ -69,9 +70,13 @@ jobs:
         id: validate
         run: terraform validate -no-color
 
+      - name: Terraform Refresh
+        id: refresh
+        run: terraform refresh
+
       - name: Terraform Plan
         id: plan
-        run: terraform plan -input=false -no-color
+        run: terraform plan -refresh=false -input=false -no-color
```

Comment:
Why the need for `terraform refresh` according to the TF documentation it says that this command is deprecated and not needed. 

https://www.terraform.io/docs/cli/commands/refresh.html

Reply:
Well dang... didn't notice it was deprecated.  The reason I added it was to remove the refresh output from the `tf plan`, which makes it easier to read (and keeps us below that ~65,000 character limit).

I'll revert the change.